package.json

```diff
@@ -3,10 +3,10 @@
   "version": "1.0.0",
   "private": true,
   "dependencies": {
-    "@databraid/github-widget": "^1.0.23",
-    "@databraid/sheets-widget": "^1.0.5",
-    "@databraid/slack-widget": "^1.0.24",
-    "@databraid/transit-widget": "^1.0.26",
+    "@databraid/github-widget": "^1.0.30",
+    "@databraid/slack-widget": "^1.0.29",
+    "@databraid/transit-widget": "^1.0.29",
+    "@gdatabraid/sheets-widget2": "^1.0.3",
```

Comment:
I'd really like to see the sheets-widget package restored so it looks like the other ones. Maybe we can get josh or hamid to give us access to databraid. 

One other possibility I have in mind is to remake all the packages not just sheets under the organization @databraid-bashboard on npm, since that in the name of the org on GitHub. Just my $0.02

Reply:
yes sir, it would be nice! wasn't that hard to do, i'm just focused on deployment and also having another person clone and run locally for now.